packages/lodestar/src/metrics/validatorMonitor.ts

```diff
@@ -204,8 +212,26 @@ export function createValidatorMonitor(
         } else {
           metrics.validatorMonitor.prevEpochOnChainTargetAttesterMiss.inc({index});
         }
-        if (summary.inclusionDistance) {
-          metrics.validatorMonitor.prevEpochOnChainInclusionDistance.set({index}, summary.inclusionDistance);
+        const prevEpochSummary = monitoredValidator.summaries.get(previousEpoch);
+        const attestationCorrectHead = prevEpochSummary?.attestationCorrectHead;
+        if (attestationCorrectHead !== null && attestationCorrectHead !== undefined) {
+          metrics.validatorMonitor.prevOnChainAttesterCorrectHead.set({index}, attestationCorrectHead ? 1 : 0);
+        }
+        const attestationMinBlockInclusionDistance = prevEpochSummary?.attestationMinBlockInclusionDistance;
+        let inclusionDistance;
+        if (
+          attestationMinBlockInclusionDistance !== undefined &&
+          attestationMinBlockInclusionDistance !== null &&
```

Comment:
You can do `attestationMinBlockInclusionDistance != null` (with single `=`) to check for undefined and null in one statement